Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.xaml

```diff
@@ -35,13 +33,8 @@
         <Setter Property="Template" Value="{StaticResource MSEdgeNotificationTemplate}" />
     </Style>
 
-    <contract7NotPresent:Style BasedOn="{StaticResource BaseInAppNotificationsStyle}"
-                               TargetType="local:InAppNotification">
-        <Setter Property="Background" Value="{ThemeResource SystemControlBackgroundChromeMediumLowBrush}" />
-    </contract7NotPresent:Style>
-
-    <contract7Present:Style BasedOn="{StaticResource BaseInAppNotificationsStyle}"
+    <Style BasedOn="{StaticResource BaseInAppNotificationsStyle}"
                             TargetType="local:InAppNotification">
         <Setter Property="Background" Value="{ThemeResource SystemControlChromeMediumLowAcrylicElementMediumBrush}" />
-    </contract7Present:Style>
+    </Style>
```

Comment:
The Background setter should be able to be part of the main style above now, it was only pulled out because it needed to be different between the two different versions. We should still have the named style and the implicit style based on construct though.

```xml
    <Style BasedOn="{StaticResource BaseInAppNotificationStyle}" TargetType="local:InAppNotification"/>
```

Reply:
Fixed on https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3478/commits/4e42672c108ff70572657be70116492f885fd336